### PR TITLE
'reset_password'

### PR DESCRIPTION
**Pull request by parsa-rahbari-82**
reset password urls and view added

### CODE REVIEW

File: account/urls.py [hunk 1]
```diff
@@ -3,5 +3,9 @@
 app_name = 'account'
 urlpatterns = (
     path('login/', views.Login.as_view(), nane='login'),
-    path('register/', views.Register.as_view(), name='register')
+    path('register/', views.Register.as_view(), name='register'),
+    path('reset/', views.UserPassReset.as_view(), name='reset_pass'),
```
Comment (RealNitroZeus) [inline account/urls.py:7]: Change it to reset-password

File: account/urls.py [hunk 2]
```diff
@@ -3,5 +3,9 @@
...
+    path('reset/done/', views.PasswordResetDone.as_view(), name='password_reset_done'),
```
Comment (RealNitroZeus) [inline account/urls.py:8]: Change it to reset-password-done

File: account/urls.py [hunk 3]
```diff
@@ -3,5 +3,9 @@
...
+    path('confirm/<uidb64>/<token>/', views.PasswordResetConfirm.as_view(), name='password_reset_confirm'),
```
Comment (RealNitroZeus) [inline account/urls.py:9]: Change it to reset-confirm

File: account/urls.py [hunk 4]
```diff
@@ -3,5 +3,9 @@
...
+    path('confirm/done/', views.PasswordResetComplete.as_view(), name='password_reset_complete'),
```
Comment (RealNitroZeus) [inline account/urls.py:10]: Change it to reset-confirm-done

Comment (parsa-rahbari-82) [reply account/urls.py:10]: i think the name of them are good and it is not important

**Pull request merged**